src/driver/gen_rpc_driver_ssl.erl

```diff
@@ -37,6 +35,46 @@
         set_send_timeout/2,
         set_acceptor_opts/1]).
 
+%%% Default SSL options common to client and server
+-define(SSL_DEFAULT_COMMON_OPTS, [binary,
+        {packet,0},
+        {header,0},
+        {exit_on_close,true},
+        {nodelay,true}, % Send our requests immediately
+        {send_timeout_close,true}, % When the socket times out, close the connection
+        {delay_send,false}, % Scheduler should favor timely delivery
+        {linger,{true,2}}, % Allow the socket to flush outgoing data for 2" before closing it - useful for casts
+        {reuseaddr,true}, % Reuse local port numbers
+        {keepalive,true}, % Keep our channel open
+        {tos,72}, % Deliver immediately
+        {active,false},
+        %% SSL options
+        {ciphers,["ECDHE-ECDSA-AES256-GCM-SHA384","ECDHE-RSA-AES256-GCM-SHA384",
+                  "ECDHE-ECDSA-AES256-SHA384","ECDHE-RSA-AES256-SHA384","ECDHE-ECDSA-DES-CBC3-SHA",
+                  "ECDH-ECDSA-AES256-GCM-SHA384","ECDH-RSA-AES256-GCM-SHA384","ECDH-ECDSA-AES256-SHA384",
+                  "ECDH-RSA-AES256-SHA384","DHE-DSS-AES256-GCM-SHA384","DHE-DSS-AES256-SHA256",
+                  "AES256-GCM-SHA384","AES256-SHA256","ECDHE-ECDSA-AES128-GCM-SHA256",
+                  "ECDHE-RSA-AES128-GCM-SHA256","ECDHE-ECDSA-AES128-SHA256","ECDHE-RSA-AES128-SHA256",
+                  "ECDH-ECDSA-AES128-GCM-SHA256","ECDH-RSA-AES128-GCM-SHA256","ECDH-ECDSA-AES128-SHA256",
+                  "ECDH-RSA-AES128-SHA256","DHE-DSS-AES128-GCM-SHA256","DHE-DSS-AES128-SHA256","AES128-GCM-SHA256",
+                  "AES128-SHA256","ECDHE-ECDSA-AES256-SHA","ECDHE-RSA-AES256-SHA","DHE-DSS-AES256-SHA",
+                  "ECDH-ECDSA-AES256-SHA","ECDH-RSA-AES256-SHA","AES256-SHA","ECDHE-ECDSA-AES128-SHA",
+                  "ECDHE-RSA-AES128-SHA","DHE-DSS-AES128-SHA","ECDH-ECDSA-AES128-SHA","ECDH-RSA-AES128-SHA","AES128-SHA"]},
+        {secure_renegotiate,true},
+        {reuse_sessions,true},
+        {versions,['tlsv1.2','tlsv1.1']},
```

Comment:
^ TLS v1.1? Mmmmh...

The list of `ciphers` also smells - these things tend to get outdated quickly when unmaintained, allowing for the use of vulnerable ciphers.

I'll need to investigate what's our best option here. Either
- drop both `ciphers` and `versions` and leave it to OTP to decide (might this risk insecure setups?)
- check what are the latest recommendations from some reliable authority (e.g. Mozilla) and change `ciphers` and `versions` accordingly

Reply:
Yeah, I just moved code around, here; didn't try to innovate. You might want to do that, but I propose doing it in a different pull request (you might want to create a task, for starters).

Reply:
Wouldn't it count as _modernizing_? 🙂

Reply:
> Wouldn't it count as _modernizing_ 

Not for me 😄. It's working. You can push on top of my changes once I'm done, though, but I don't want to put everything into a single pull request: it increases the chances of "never-endingness".

Reply:
(I will, however, dedicate some minutes to this - if I'm unable to find the info. I'm looking for, I'll leave this juice for you)

Reply:
There you go: 044fb46.

Source: https://wiki.mozilla.org/Security/Server_Side_TLS

Reply:
Very nice, much secure